packages/circuit-ui/components/ImageInput/ImageInput.docs.mdx

```diff
@@ -49,3 +49,17 @@ There are some caveats with this approach:
 ### Using a div element
 
 <Story id="forms-imageinput--custom-component-div" />
+
+## Accessibility
```

Comment:
I'm noticing other issues when using the ImageInput with Firefox and VoiceOver. Can you share your findings from SR testing so we can compare what we found and address any problems?

Reply:
Moved it to polite after testing in isolation on storybook with Firefox,Chrome and VoiceOver. I will add recordings as reference. Let me know if you have any more findings.

Reply:
Attaching here the recording with Firefox and VoiceOver. For some reason after my Chrome update the screen reader stopped announcing message completely when I tested in isolation. Let me know if you can try on your side and let me know the result.

https://user-images.githubusercontent.com/57903317/179235726-3f9bf811-5866-49b7-9ab4-3baa857889cf.mov

.

Reply:
We can sync on later then on this and the docs.